### ISSUE TITLE
Adding custom http api authorizer

### ISSUE THREAD

**Issue by dapriett** [serverless-components/express#19]
Greetings.

Does the express component support API Gateway HTTP API [authorizers](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-jwt-authorizer.html)?  Or if not, how best to inject one in?

Thank you!

**Comment by eahefnawy** [comment]
Nope, not at the moment at least. The express component is designed with a single mono lambda. You could put your authorization logic there like you would with any express app.
